config/jest/test.config.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * This is the main jest config.  It runs tests using the default
+ * test environment: jest-environment-node.
+ */
+const path = require("path");
+
+module.exports = {
+    rootDir: path.join(__dirname, "../../"),
+    transform: {
+        "^.+\\.jsx?$": "<rootDir>/config/jest/test.transform.js",
+    },
+    testEnvironment: "jest-environment-node",
+    testMatch: ["<rootDir>/**/*.test.js"],
+    setupFilesAfterEnv: ["<rootDir>/config/jest/test-setup.js"],
+    moduleNameMapper: {
+        "^@khanacademy/wonder-stuff-(.*)$":
+            "<rootDir>/packages/wonder-stuff-$1/src/index.js",
+    },
+    collectCoverageFrom: [
+        "packages/**/*.js",
+        "!packages/**/*.flowtest.js",
+        "!packages/**/dist/**/*.js",
+        "!<rootDir>/node_modules/",
+        "!packages/**/node_modules/",
```

Comment:
Instead of collecting coverage from `packages/**/*.js` and then having to exclude node_modules, we should probably collect coverage on `packages/*/src/**/*.js`.  Having all of the source files live in `packages/*/src` will also make migrating to TypeScript easier because it makes using project reference and incremental compilation easier because `out/` can be a sibling of `src/`.  In this situation `out/` is separate from `dist/` and acts more like the babel cache.

Reply:
I see we're putting source files in `packages/*/src` in #4 so this is no longer blocking but more of possible tweak.